presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4531,6 +4532,23 @@ public void testSemiJoinNullHandling()
     }
 
     @Test
+    public void testScalarSubquery()
```

Comment:
We're probably going to need more tests.
* multiple scalar subqueries in a single query
* combination of IN predicate and scalar subqueries in the same query
* combinations with other features (joins -- all types, distinct, etc)